language/extensions/move-table-extension/sources/Table.move

```diff
@@ -0,0 +1,93 @@
+/// Type of large-scale storage tables.
+module Extensions::Table {
+    /// Type of tables
+    struct Table<phantom K, phantom V> has store {
```

Comment:
Is see you have `V: store` on `new` so we should be fine, but should also have `V: store` on the declaration?

I understand it isn't necessary, but it might be less confusing (that is `struct Table<phantom K, phantom V: store> has store`)

Reply:
Done.